components/symbl_ai/actions/get-action-items/get-action-items.mjs

```diff
@@ -0,0 +1,30 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-action-items",
+  name: "Get Action Items",
+  description: "Get a list of all the action items generated from the conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/action-items)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      type: "string",
+      label: "Conversation Id",
+      description: "The Id of the Conversation",
+    },
+  },
+  async run({ $ }) {
+    try {
+      const response = await this.symblAIApp.getActionItems({
+        $,
+        conversationId: this.conversationId,
+      });
+      $.export("$summary", "Action Items successfully retrieved from the conversation");
+      return response;
```

Comment:
**Suggestion** (for this action and the other new actions)

The return value from a "list" action is typically an array of the items being listed, to simplify using the results from actions in workflows. Some examples in existing components: [Airtable](https://github.com/PipedreamHQ/pipedream/blob/78b07a8bc24adb1724467987e85743fa7bed711c/components/airtable/actions/common-list.js#L54), [Google Sheets](https://github.com/PipedreamHQ/pipedream/blob/78b07a8bc24adb1724467987e85743fa7bed711c/components/google_sheets/actions/list-worksheets/list-worksheets.js#L30), [Eventbrite](https://github.com/PipedreamHQ/pipedream/blob/78b07a8bc24adb1724467987e85743fa7bed711c/components/eventbrite/actions/get-event-attendees/get-event-attendees.js#L41)

Currently the array of `actionsItems` is nested in the return value (`{"$return_value":{"actionItems":[]}}`).

```suggestion
      const { actionItems } = await this.symblAIApp.getActionItems({
        $,
        conversationId: this.conversationId,
      });
      $.export("$summary", "Action Items successfully retrieved from the conversation");
      return actionItems;
```

Reply:
Fixed in abb1fb0d